speech/tests/gapic/test_speech_client_v1.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests."""
+
+import mock
```

Comment:
Code generator is fake news! `mock` isn't stdlib (at least not in Python 2, and in Python 3 it's found as `unittest.mock`)